pkg/control/v2/server/server.go

```diff
@@ -138,12 +136,16 @@ func (s *Server) State(_ context.Context, _ *cproto.Empty) (*cproto.StateRespons
 // StateWatch streams the current state of the Elastic Agent to the client.
 func (s *Server) StateWatch(_ *cproto.Empty, srv cproto.ElasticAgentControl_StateWatchServer) error {
 	ctx := srv.Context()
-	sub := s.coord.StateSubscribe(ctx)
+	// TODO: Should we expose the subscription buffer size in the RPC? This
+	// would e.g. let subscribers who only care about the latest state set a
+	// buffer size of 0 so they will always receive the most recent value
+	// instead of the full sequence.
```

Comment:
+1 to exposing it. I believe it is completely possible that some subscribers only want the latest.